### PR TITLE
15.0 mig partner contact department

### PR DESCRIPTION
**Pull request by Galoperd**

### CODE REVIEW

File: partner_contact_department/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    department_id = fields.Many2one("res.partner.department", "Department")
+
+
+class ResPartnerDepartment(models.Model):
```
Comment (rousseldenis) [inline partner_contact_department/models/res_partner.py]: Please put this model in separate file.

Comment (Galoperd) [reply partner_contact_department/models/res_partner.py]: @rousseldenis done

Comment (Galoperd) [reply partner_contact_department/models/res_partner.py]: please note there is a mistake with your module naming: /ocabot migration contact_partner_department
it should be /ocabot migration partner_contact_department, so there is a dupplicated entry at [migration list](https://github.com/OCA/partner-contact/issues/1153)

@rousseldenis please kindly check, thank you !

**Comment by Galoperd** [comment]
@rousseldenis many thanks for solving this. No duplicate work existing so this [PR#1322](https://github.com/OCA/partner-contact/pull/1322) is ready for review and merge.

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by andres-pcg** [comment]
@rousseldenis is there any reason why runboat/build is still pending? I believe that's the only thing that's missing to have this merged. 

Regards,

**Comment by Galoperd** [comment]
@rousseldenis not sure why runboat/build was not auto processed… is there something wrong in the code or do I need to do something else ? thanks

**Comment by rousseldenis** [comment]
> @rousseldenis not sure why runboat/build was not auto processed… is there something wrong in the code or do I need to do something else ? thanks

I think runboat was sick at that time.

Could you try to rebase?

**Comment by andres-pcg** [comment]
@Galoperd Thanks for the follow-up. Tests are now failing 😥

**Pull request closed without merging**